proto/v1alpha2/api.proto

```diff
@@ -96,14 +96,14 @@ service Results {
 service Logs {
   rpc GetLog(GetLogRequest) returns (stream Log) {
     option (google.api.http) = {
-      get: "/v1alpha2/{name=*/results/*/logs/*}"
+      get: "/v1alpha2/logs/{name=*/results/*/records/*}"
     };
```

Comment:
Please add `results` prefix for Results API.

Also, won't it be better to have rest APIs like below: 
`/apis/results.tekton.dev/v1alpha2/logs`

Reply:
Right.. I think this `/apis/results.tekton.dev/v1alpha2/logs` pattern is better. This follows the kube api access pattern. 
I'll change this for all the RPCs

Reply:
+1